homeassistant/components/tautulli/coordinator.py

```diff
@@ -44,9 +44,9 @@ def __init__(
         )
         self.host_configuration = host_configuration
         self.api_client = api_client
-        self.activity: PyTautulliApiActivity | None = None
-        self.home_stats: list[PyTautulliApiHomeStats] | None = None
-        self.users: list[PyTautulliApiUser] | None = None
+        self.activity: PyTautulliApiActivity = PyTautulliApiActivity({"": ""})
+        self.home_stats: list[PyTautulliApiHomeStats] = []
+        self.users: list[PyTautulliApiUser] = []
```

Comment:
These should all be able to be None and should be initialized with None, why is this changed?